Formula/ntfy.rb

```diff
@@ -0,0 +1,44 @@
+class Ntfy < Formula
+  desc "Send push notifications to your phone or desktop via PUT/POST"
+  homepage "https://ntfy.sh/"
+  url "https://github.com/binwiederhier/ntfy.git",
+      tag: "v2.3.1",
+      revision: "a75fb08ef138d67599499c1f78628a0b35fcef54"
```

Comment:
```suggestion
      tag:      "v2.3.1",
      revision: "a75fb08ef138d67599499c1f78628a0b35fcef54"
```
Forgot about this one. This one is part of a style check.

Reply:
yep, fixing